_locales/ca/messages.json

```diff
@@ -547,6 +547,22 @@
         "message": "When this key is held down, the picture is displayed using all available space. Useful for high resolution pictures only.",
         "description": "[options] Action key description"
     },
+    "optCopyImageKeyTitle": {
```

Comment:
There's no need to update any localization files other than `en`, it will be taken care of automatically through https://crowdin.com/project/hoverzoom integration.

Reply:
Ah that makes sense. I've reverted the `ca` and `uk` locale files.